xmake/rules/c++/modules/build_modulefiles.lua

```diff
@@ -35,7 +35,7 @@ function _build_modulefiles_clang(target, sourcebatch, opt)
     end
 
     -- compile module files to *.pcm
-    opt = table.join(opt, {configs = {force = {cxxflags = {"-fmodules-ts", "--precompile", "-x c++-module"}}}})
+    opt = table.join(opt, {configs = {force = {cxxflags = {modulesflag, "--precompile", "-x c++-module"}}}})
```

Comment:
opt.modulesflag

Reply:
Changed the code above with `opt.modulesflag` and will post it later.